viya4-deployment-darksite/baseline-to-ecr/00_vars.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Do the .sh files need a copyright header?

Reply:
good catch Jay, will add.